Packages/vcs/Lib/VTKPlots.py

```diff
@@ -913,13 +915,13 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
               tprops.AddItem(tprop)
               if colorOverride is not None:
                   del(vcs.elements["texttable"][tt])
-
-              mapper.SetTextProperties(tprops)
+          mapper.SetTextProperties(tprops)
          else:
             # No text properties specified. Use the default:
-            tprop = vtk.vtkTextProperty()
-            vcs2vtk.prepTextProperty(tprop, self.renWin.GetSize())
-            mapper.SetTextProperty(tprop)
+            tprops = vtk.vtkTextProperty()
```

Comment:
This variable name should to be changed back -- it's only a single text property object.